src/Pages/Dashboard.js

```diff
@@ -1,26 +1,28 @@
 import React, { Component } from 'react';
 import AuthService from '../services';
-import My_Task_Card from './Components/My_task_card';
+import My_Task_Card from '../Components/My_task_card';
```

Comment:
@l1ngoman @jscotty723

This is breaking the react conventions of CamelCasing your component names.   Engineers will notice this.